[2013-09-02 14:28:20] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429
Stack trace:
#0 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\vhosts\laraveil\public\index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-09-02 14:28:41] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429
Stack trace:
#0 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\vhosts\laraveil\public\index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-09-02 14:28:58] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429
Stack trace:
#0 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\vhosts\laraveil\public\index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-09-02 17:25:21] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'User' not found' in D:\vhosts\laraveil\app\routes.php:19
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
